tools/BUILD.bazel

```diff
@@ -32,13 +33,23 @@ generate_test(
 
 format(
     name = "buildifier_format",
-    srcs = ["@rules_file_files//:buildifier_files"],
+    srcs = ["@files//:buildifier_files"],
     formatter = ":buildifier",
     strip_prefix = "/files",
 )
 
-sh_binary(
+find_packages(
+    name = "test_packages",
+    roots = [
+        "file/test/bazel",
+    ],
+)
+
+bazelrc_deleted_packages(
```

Comment:
See (self-hosting) usage here

Reply:
A good idea. I've thought about that.

It's going to use something different than the rules_file generate framework, because it needs to access all sources in BUILD_WORKSPACE_DIRECTORY.